lib/src/source/git.dart

```diff
@@ -254,7 +254,7 @@ class GitSource extends CachedSource {
       fail('Could not find a file named "$pathInCache" in '
           '${GitDescription.prettyUri(description.url)} $revision.');
     }
-    return lines.join();
```

Comment:
Would it be wiser to make a temporary file and do `--output=tempFile` ?

Reply:
I think we should just not split the output in the first place. But let's do it later.